x-pack/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -894,38 +889,6 @@ export default function ({ getService }: FtrProviderContext) {
       });
     });
 
-    it('should allow a failed task to be rerun using runSoon', async () => {
```

Comment:
Allowing failed ad-hoc tasks to be re-run was a feature that was never used, so it no longer works after this PR.